tools/image_processing/imagej2/imagej2_noise/imagej2_noise.py

```diff
@@ -0,0 +1,100 @@
+# Galaxy wrapper for use with Imagej2 by Greg Von Kuster
+"""
```

Comment:
This is totally fine with me. I also don't know if this will work out, so I like this idea. Just wanted to point out this other possibility.